packages/core/src/chains/sol/sdkSol/OffchainMessage.ts

```
@@ -0,0 +1,147 @@
+                return 2;
+            }
+        }
+        return undefined;
+    }
+
+    static isPrintableASCII(buffer: Buffer) {
+        return (
+            buffer &&
+            buffer.every((element) => {
+                return element >= 0x20 && element <= 0x7e;
+            })
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)